xmtp_dh/Cargo.toml

```diff
@@ -18,10 +18,14 @@ strip = true        # Strip symbols from binary*
 [lib]
 crate-type = ["cdylib"]
 
+[[bin]]
+name = "xmtp-dh-gen"
+path = "src/bin.rs"
+
 [dependencies]
-uniffi = { version = "0.23.0" }
-uniffi_macros = "0.23.0"
+uniffi = { git = "https://github.com/mozilla/uniffi-rs", rev = "755049ebaa967fdfbb192d5b9c111353a7fee99f", features = ["tokio", "cli"] }
```

Comment:
Would it be possible to track this version information in the workspace a la :https://github.com/rust-lang/cargo/issues/3931?

Worried managing the git refs might get annoying